README.md

```diff
@@ -122,6 +122,8 @@ To build locally, run `make build` (see below).
 
 ## Other examples
 
+![image](https://user-images.githubusercontent.com/42107491/235344183-44b5967d-ba22-4331-804c-8da7004a5d35.png)
```

Comment:
maybe all the images should go in the index after all, I'd just leave here a simple call out link to the example sections